packages/docusaurus/src/server/presets/index.ts

```diff
@@ -7,15 +7,20 @@
 
 import importFresh from 'import-fresh';
 import _ from 'lodash';
-import {LoadContext, PluginConfig, Preset, PresetConfig} from '../types';
+import {
+  LoadContext,
+  PluginConfig,
+  Preset,
+  PresetConfig,
+} from '@docusaurus/types';
 
 export function loadPresets(
-  context: LoadContext,
+  context: Partial<LoadContext>,
```

Comment:
does it have to be partial ?

Reply:
Yes, `loadPresets` method is needed some of `LoadContext`.